app/pods/cases/index/route.js

```diff
@@ -22,15 +24,22 @@ export default class OverviewCaseRoute extends Route.extend(DataTableRouteMixin)
     },
   };
 
-  modelName = 'case';
-
-  // eslint-disable-next-line class-methods-use-this
-  mergeQueryOptions(params) {
+  async model(params) {
+    const options = {
+      sort: params.sort,
+      page: {
+        number: params.page,
+        size: params.size,
+      },
+    };
+    if (params.filter) {
+      options.filter = params.filter;
+    }
     if (!params.showArchived) {
-      return {
-        'filter[is-archived]': false,
-      };
+      options['filter[is-archived]'] = false;
     }
+    const model = await this.store.query('case', options);
+    return model;
```

Comment:
Kleine optimalisatie: als de `model`-hook een promise teruggeeft, wordt daar automatisch op gewacht. Je kan dit dus ook korter schrijven als `return this.store.query('case', options)` waardoor de model-hook niet meer als `async` gemarkeerd moet worden (maar wat er staat is op zich dus niet fout)